core/database/build.gradle.kts

```diff
@@ -34,6 +34,18 @@ android {
         testInstrumentationRunner = "com.google.samples.apps.nowinandroid.core.testing.NiaTestRunner"
     }
     namespace = "com.google.samples.apps.nowinandroid.core.database"
+
+    testOptions {
```

Comment:
Can we move this into a common plugin so that GMD is used for every module?

Reply:
There was a bug where defining GMD wasn't possible in Kotlin that made impossible to define it as a convention plugin. (b/242515648)

That is now fixed, but is included in Flamingo canary 9.

Reply:
I was going to suggest the same, thanks for the info. Can we create a TODO and a GitHub issue to track?